.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby_version: [2.5.x, 2.6.x, 2.7.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby_version }}
+    - name: Calculate variable dynamic values
+      id: dynamic_values
+      run: |
+        echo "::set-output name=installed_ruby_version::$(ruby -e 'print RUBY_VERSION')"
+        echo "::set-output name=cacheTimeAnchor::$(ruby -e 'require %Q{date}; cacheExpirationSeconds = 60*60*24; print (Time.now.to_i / cacheExpirationSeconds)')"
+    - name: Download CodeClimate reporter
+      run: |
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        chmod +x ./cc-test-reporter
+        ./cc-test-reporter before-build
+      env:
+        CC_TEST_REPORTER_ID: 903dc1a761aaec438bc9f39467e2cb4a2ea332bdb65241b37abef6338c5e6326
+    - name: Install and config bundler
+      run: |
+        gem install bundler:1.17.3
```

Comment:
Perhaps we can use 2.1.4, WDYT?

Reply:
Sure! This file was autogenerated by [rsgem](https://github.com/rootstrap/rsgem/blob/master/.github/workflows/ci.yml#L32), do you think we can open an issue to update this version there too?

Reply:
Sure thing!